<launch>
    <node
        pkg="robotics_cswk_kin"
        type="inv_kinematics_service.py"
        name="inv_kin_service"
        output="screen"
    />
</launch>